.github/workflows/Pipeline.yml

```
@@ -27,12 +27,13 @@ jobs:
 
           - qemu: arm
             container: arm32v7
-            # Note: test 'load_store' is disabled because arm on qemu-user-static seems not to support '-mfpu=neon'
+            # Note that the following tests are disabled on arm32v7:
+            # - load_store: arm on qemu-user-static seems not to support '-mfpu=neon'
+            # - signals: when running on MAMBO through qemu-user-static it hangs frequently
             tests: >-
               mmap_munmap
               mprotect_exec
               self_modifying
-              signals
 
```

The `load_store` should be added.

```suggestion
load_store
```